tools/doc-generator/parse/parser.go

```diff
@@ -449,32 +452,6 @@ func getFieldType(t reflect.Type) (string, error) {
 	}
 }
 
-func getCustomFieldType(t reflect.Type) (string, bool) {
```

Comment:
Wow, funny that we had a duplicate function of `getFieldCustomType`. Good catch!